src/compiler/transformers/module/module.ts

```diff
@@ -745,11 +745,20 @@ namespace ts {
                     else if (declaration.kind === SyntaxKind.ImportSpecifier) {
                         const name = (<ImportSpecifier>declaration).propertyName
                             || (<ImportSpecifier>declaration).name;
-                        return createPropertyAccess(
-                            getGeneratedNameForNode(declaration.parent.parent.parent),
-                            getSynthesizedClone(name),
-                            /*location*/ node
-                        );
+                        if (name.originalKeywordKind === SyntaxKind.DefaultKeyword && languageVersion <= ScriptTarget.ES3) {

```

Comment:
`default` is not necessarily the only thing that needs an element access in ES3. ES3 permits *IdentifierName*s but not *ReservedWord*s to be on the right of a property access.